app/db/queries.py

```diff
@@ -5,44 +5,62 @@
 
 
 def get_messages_that_contain_word(word: str) -> List[Message]:
-    pass
+    return list(Message.objects.filter(text__icontains=word))
 
 
 def get_untitled_chats() -> List[Chat]:
-    pass
+    return list(Chat.objects.filter(title__istartswith="Untitled"))
 
 
 def get_users_who_sent_messages_in_2015() -> List[str]:
-    pass
+    return list(Message.objects.filter(
+        sent__year=2015).values_list("user__first_name", "user__last_name"))
 
 
 def get_actual_chats() -> List[Chat]:
-    pass
+    return list(Chat.objects.filter(
+        message__sent__year__gte=2020).distinct())
 
 
 def get_messages_contain_authors_first_name():
-    pass
+    return list(Message.objects.filter(
+        text__icontains=F("user__first_name")))
 
 
 def get_users_who_sent_messages_starts_with_m_or_a() -> List[User]:
-    pass
+    return list(User.objects.filter(
+        Q(message__text__istartswith="m") | Q(message__text__istartswith="a")))
 
 
 def get_delivered_or_admin_messages() -> List[Message]:
-    pass
+    return list(Message.objects.filter(
+        Q(is_delivered=True) | Q(user__username__istartswith="admin")))
 
 
 def get_count_messages_sent_by_first_name(first_name: str) -> int:
-    pass
+    return Message.objects.filter(
+        user__first_name=first_name).count()
 
 
 def get_top_users_by_number_of_the_messages() -> List[User]:
-    pass
+    return User.objects.annotate(
+        num_messages=Count("message")).order_by("-num_messages")[:3]
 
 
 def get_last_5_messages_dicts() -> List[dict]:
-    pass
+    last_5_messages = []
+    for message in \
+            Message.objects.all().select_related("user").order_by("-sent")[:5]:
```

Comment:
Here creating a new variable for the list of the massages would increase readability. 
This is a bit different situation than in previous functions because should not just return a list but iterate through it.